stubs/EloquentBuilder.stub

```diff
@@ -145,6 +145,14 @@ class Builder
      */
     public function firstOrFail($columns = ['*']);
 
+    /**
+     * Execute the query and get the first result if it's the sole matching record.
+     *
+     * @param  array<int, (model-property<TModelClass>|'*')>|model-property<TModelClass>|'*'  $columns
+     * @phpstan-return TModelClass
+     */
```

Comment:
Now that the other PR is merged, can you add back the `@throw` tags again here?

Reply:
Not sure that I can but I will try…

Reply:
Why not?